examples/langchain/dkg-demo.js

```diff
@@ -0,0 +1,20 @@
+import "dotenv/config";
+import DKG from "dkg.js";
+import fs from "fs";
+
+const yewmakerol = JSON.parse(fs.readFileSync("./yewmakerol.json"));
+
+const dkg = new DKG({
+  endpoint: process.env.OT_NODE_HOSTNAME,
+  blockchain: {
+    name: "otp::testnet",
```

Comment:
I would put into readme link to our discord link and say that they can get tokens from our faucet on discord